### DISCUSSION TITLE
Feedback/requests

### DISCUSSION THREAD

**Discussion by Fictitiousfreedom** [keiwando/toonsquid#290] [Ideas]
I absolutely love toonsquid, its such a huge step in the right direction, nothing else even compares. I mostly record music but started animation as a way to present my music which is mostly instrumental. Ive used every ipad app and use toonboom on my mac. Even though toonboom is incredible i find myself wanting to just use my ipad for everything. Its much more intuitive. Every app ive tried is pretty bad, until now. 
This has all possibly been discussed but incase it has not here are the things that would launch toonsquid into another level that seem to me easily implemented, but idk shit lol

1. saving symbols and reusing them. This is a must. 
2. a way to use keyframes on groups to overwrite, copy, and paste everything. Having to go into each piece of my rig and copy and paste all of the keys takes an insane amount of time. For example in toonboom if the rig or parts of the rigs layers are collapsed you can write keyframes for every part of that group at once.  Also a way to reset all keys or parts to a default position is absolutely necessary. Everything becomes so messy and impossible to use on semi complex rigs. 
3. A way to turn off motion paths, its very annoying when you go to transform a part and it grabs the motion path instead. 

Those are the most important to me. Thank you for this app, your truly a genius mind. The entire concept of this app is so incredibly innovative and i support you all the way.

**Comment by keiwando** [comment]
Thanks for the feedback and the kind words!
